src/Workspaces/Core/Portable/NamingStyles/NamingStyleRules.cs

```diff
@@ -46,5 +62,33 @@ private bool IsSymbolNameAnalyzable(ISymbol symbol)
 
             return true;
         }
+
+        private bool IsInterfaceImplementation(ISymbol symbol)
+        {
+            var containingType = symbol.ContainingType;
+            if (containingType == null)
```

Comment:
technically, this can't ever happen.  All methods/properties/events must always have a containing type.